source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -872,7 +872,7 @@ SwiftLanguage::GetHardcodedSummaries() {
         return swift_metatype_summary_sp;
       }
       if (valobj.GetName().GetLength() > 12 &&
-          valobj.GetName().GetStringRef().startswith("$swift.type.") &&
+          valobj.GetName().GetStringRef().startswith(u8"$\u03C4_") &&
```

Comment:
Would be nice to store the unicode string to a variable, or add a comment for posterity